Lambdas/GHOAuth/Models/GHOAuthPayload.swift

```diff
@@ -1,5 +1,6 @@
 import DiscordBM
 import JWTKit
+import struct Foundation.Date
```

Comment:
Shouldn't this be `FoundationEssentials.Date` to avoid linking all of Foundation?